File: C_Programs/Print1To10UsingDoWhileLoop.cpp
// Print1To10UsingDoWhileLoop.cpp : This file contains the 'main' function. Program execution begins and ends there.
//

#include <iostream>

int main()
{
	int num=1;
	do
	{
		printf("%d\t", num);
		num++;
	} while (num <= 10);
	return 0;
}

File: Core_Java_Programs/Print1To10UsingDoWhileLoop.java
//print 1 to 10 using while loop
import java.util.*;

public class Print1To10UsingDoWhileLoop {
    public static void main(String[] args) {
        int num = 1;
        do {
            System.out.print(num + "\t");
            num++;
        }while(num<=10);
    }
}